### PR TITLE
Feature/pvp hooks

### PR DESCRIPTION
**Pull request by loganlearner**
TEST CODE:
```lua
--@shared

hook.add("playerExitPvp", "test1", function(ply)
    print(ply)
end)

hook.add("playerEnterPvp", "test2", function(ply)
    print(ply)
end)
```

OUTPUT:
image

### CODE REVIEW

File: lua/starfall/libs_sh/pvp.lua [hunk 1]
```diff
@@ -69,7 +60,7 @@ function pvp_library.playerIsInPvp( ply )
     return getPly( ply ):isInPvp()
 end
 
---- Returns true if the player is in Pvp mode, false otherwise.	
+--- Returns true if the player is in Pvp mode, false otherwise.	=
```
Comment (wrefgtzweve) [inline lua/starfall/libs_sh/pvp.lua:63]: Unacceptable

Comment (brandonsturgeon) [inline lua/starfall/libs_sh/pvp.lua:63]:
```suggestion
--- Returns true if the player is in Pvp mode, false otherwise.
```

**Review by brandonsturgeon — APPROVED** [review]

**Comment by wrefgtzweve** [comment]
I'd assume SF hook add automatically wraps it's outputs so lgtm

**Comment by ProfessorBoxtrot** [comment]
all SF hooks/methods aren't scoped (CFC_Pvp_), it seems weird to have this be different imo

**Comment by brandonsturgeon** [comment]
Well that's the name of the actual hook. Can you alias hook names in Starfall?

**Review by brandonsturgeon — CHANGES_REQUESTED** [review]
@ProfessorBoxtrot made a good point, we should probably make these hook names consistent with starfall syntax

**Comment by loganlearner** [comment]
I'll give it one more test and merge!

**Comment by loganlearner** [comment]
Tested, working as intended. Merging!

**Pull request merged**